script/Arcueid/Day 10/10_00_ARC10_3_1.txt

```diff
@@ -276,7 +276,8 @@ So she's already at the point of 'I'm bored, show me a magic trick'.
 [sha:b9cc7321a035487e7ef180b7f8c9f327248713af]{
 -- Page 142, Offset 24618.
 -- 「んー、しりとり、とか？」
-"Hmm, how about <shiritori|w o r d   c h a i n   >?"
+// do we really need another ruby for this?
```

Comment:
shiritori is one of those things that, although it's common in anime, it's not something that I'd expect the average reader to know. "Word chain" is a nice explanation of what comes next so I think we ought to keep it.